lib/components/chat_panel.dart

```diff
@@ -54,35 +80,96 @@ class _ChatPanelWidgetState extends State<ChatPanelWidget> {
-                                        context,
-                                        listen: false);
-                                    model.delete(model.channels.first,
-                                        message.messageId);
-                                    Navigator.pop(context);
-                                  }),
-                              ListTile(
-                                  title: Text('Timeout ${message.author}'),
-                                  onTap: () {}),
-                              ListTile(
-                                  title: Text('Ban ${message.author}'),
-                                  onTap: () {}),
-                              ListTile(
-                                  title: Text('Unban ${message.author}'),
-                                  onTap: () {
-                                    final model = Provider.of<UserModel>(
-                                        context,
-                                        listen: false);
-                                    model.unban(
-                                        model.channels.first, message.author);
-                                    Navigator.pop(context);
-                                  }),
-                            ]),
-                          );
+                              shape: RoundedRectangleBorder(
+                                  borderRadius:
+                                      BorderRadius.all(Radius.circular(20.0))),
+                              child: Wrap(
+                                children: [
+                                  ListView(
+                                    shrinkWrap: true,
+                                    children: [
+                                      ListTile(
+                                        shape: RoundedRectangleBorder(
+                                            borderRadius: new BorderRadius.only(
+                                          topLeft: const Radius.circular(20.0),
+                                          topRight: const Radius.circular(20.0),
+                                        )),
+                                        leading: Icon(Icons.person,
+                                            color: Colors.pinkAccent),
+                                        title: Text('${message.author}',
+                                            style: TextStyle(fontSize: 20)),
+                                        tileColor: Colors.blueAccent,
+                                        trailing: CloseButton(),
+                                      ),
+                                      ListTile(
+                                        leading: Icon(Icons.favorite,
+                                            color: Colors.deepPurpleAccent),
+                                        title:
+                                            Text('Following Since Jan 11 2024'),
```

Comment:
Can this data be fetched from the API instead of hardcoded?

Reply:
Looks like the data can be fetched from the API, Link: [https://dev.twitch.tv/docs/api/reference#get-users-follows](https://dev.twitch.tv/docs/api/reference#get-users-follows). But I had no experience with pulling data from an API with dart.